python/lsst/meas/astrom/fitAffineWcs.py

```diff
@@ -163,52 +116,41 @@ def fitWcs(self,
         # appends the new transform.
         wcsMaker = TransformedSkyWcsMaker(initWcs)
 
-        refPoints = []
-        srcPixels = []
-        offsetLong = 0
-        offsetLat = 0
-        # Grab reference coordinates and source centroids. Compute the average
-        # direction and separation between the reference and the sources.
-        for match in matches:
+        # Grab the initial transformations going back from sky coordinates
+        # and forward from pixel coordinates.
+        back = wcsMaker.frameDict.getMapping(wcsMaker.frameMax, wcsMaker.frameMax-1)
+        forward = wcsMaker.lastMapBeforeSky
+
+        # Create containers for the data going into the Affine fit (i.e.
+        # solving Ax=b where x will be the affine parameters and a linear
+        # shift.
+        A = np.zeros((len(matches)*2, 6), dtype=float)
+        b = np.empty(len(matches)*2, dtype=float)
+
+        # Constant terms related to the shit in and and y parameters
+        A[:len(matches)*2:2, 4] = 1
+        A[1:len(matches)*2+1:2, 5] = 1
```

Comment:
I think
```
A[::2, 4] = 1
A[1::2, 5] = 1
```
is equivalent to the above, and more readable.

Reply:
I agree. Those were left from previous slicing operations when I had things arranged differently and never cleaned them up. Thanks for the detailed reading